docs/pages/setup/helm-reference/teleport-cluster.mdx

```diff
@@ -63,51 +63,82 @@ This reference details available values for the `teleport-cluster` chart.
 
 `kubeClusterName` sets the name used for the Kubernetes cluster. This name will be shown to Teleport users connecting to the cluster.
 
-## `authenticationType`
+## `authentication`
+
+### `authentication.type`
 
 | Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
 | - | - | - | - | - |
 | `string` | `local` | Yes | `auth_service.authentication.type` | ❌ |
 
-`authenticationType` controls the authentication scheme used by Teleport. Possible values are `local` and `github` for OSS, plus `oidc`, `saml`, and `false` for Enterprise.
+`authentication.type` controls the authentication scheme used by Teleport. Possible values are `local` and `github` for OSS, plus `oidc` and `saml` for Enterprise.
+
+### `authentication.connectorName`
+
+| Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
+| - | - | - | - | - |
+| `string` | `""` | No | `auth_service.authentication.connector_name` | ❌ |
+
+`authentication.connectorName` sets the default authentication connector.
+See [the SSO documentation](../../enterprise/sso.mdx). Other possible values are `local` and
```

Comment:
What are readers supposed to see in the SSO documentation? I would indicate that in this description. 

Since there are only a few SSO connectors, I think it would also make sense to list the possible values here, rather than referring the reader to another docs page.

Reply:
This field is either the authentication connector name or `local`/`passwordless`. IIUC SSO connector name != supported SSO providers. Users can give any name they want to their SSO connector and create multiple connectors for the same provider. Hence I cannot list all possible values